### PR TITLE
Refactor build process

### PR DESCRIPTION
**Pull request by lkluft**
This PR revises the build process of slides in several steps
1. Replace all `_variables_{date}.yml` files with `_metadata.yml`, which are evaluated **per directory** (no need for symlinks)
2. Remove the build script as now all slides can be easily build with a single quarto command
    ```sh
    quarto render --profile slides
    ```
3. Add a Github action that automatically builds the slides and publishes them on Github pages

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,18 +44,17 @@ this will check that all figures are complete and indicate possible missing ones
...
 for website view:
 
 ```
-./render_quarto.sh [BRIEFING_DATE] preview website
+quarto render --profile website
 ```
 
-### Build Everything
-
+You can also run quarto in preview mode, which will open a browser and navigate to the most recently modified file:
```
Comment (d70-t) [inline README.md:57]: The entire section was about preview, so this comment is a bit confusing...

**Review by d70-t — COMMENTED** [review]
Apart from minor issues with the README.md, this is a great refactoring :+1:

With the proposed changes, it's straightforward to build all presentations at once. The change also removes the risk of accidentally building the wrong slides because of an outdated symlink.

Once the comments are addressed, I'd vote for a merge sooner than later.

File: .github/workflows/static.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Setup quarto
+        uses: quarto-dev/quarto-actions/setup@v2
+        with:
+          version: 1.5.56
+      - name: Build slides
+        run: quarto render --profile slides
```
[the remark this answers is not in the document]
Comment (lkluft) [reply .github/workflows/static.yaml:39]: True! I added the build command

File: README.md (hunk 1 above)
Comment (lkluft) [reply README.md:55]: I think it was a combination of leaving the outdated subtitle und copy & pasting the last command... 😬 

My idea for the readme was to follow a _i)_ init with `wbcli`, _ii)_ render everything with quarto, _iii)_ if you feel like it use the preview structure. The updated version hopefully does that

**Comment by lkluft** [comment]
While testing the updated version of the scripts I got a bit annoyed by the warnings printed by `pandas` and `intake_xarray`, so I fixed those. I am happy to open a separate PR for this, but I would argue that this is somehow related to the build process 😉.

**Comment by sortega87** [comment]
Discussing with @HF7weatherman, our only comment would be to not activate the publishing of the reports yet, but to wait until we have a first version. Perhaps this is the case already but, in case it is not not, is it possible to do so? 

Other than that I agree with all other changes, and we could merge to main after addressing the comment. Great work! Thanks! :)

**Review by d70-t — APPROVED** [review]
Thanks for updating, looks great 👍 🎉

**Comment by lkluft** [comment]
I have checked all images that are included so far, and all of them have the watermark of their respective organisation. In addition, I added an acknowledgment section to the about page of the website.

Both briefings that are part of the repo as is are building correctly. And in case any problems occur, we can easily takedown the website. I think we could get that merged ☺️

**Review by sortega87 — APPROVED** [review]

**Pull request merged**